### ISSUE TITLE
Черный скриншот тестклиента

### ISSUE THREAD

**Issue by KrapivinAndrey** [lintest/VanessaExt#30]
1c: 8.3.17.1386
OS: ubuntu 18.04 xfce 4.12
Vanessa Ext: 1.2.2.83

Все это в docker

У одного сеанса снят скриншот, у второго нет

![image](https://user-images.githubusercontent.com/586025/97661276-bab7f780-1a95-11eb-9006-b803ced75eb7.png)

**Comment by Pr-Mex** [comment]
@KrapivinAndrey 
Это воспроизводится если подключиться в убунту и запустить тест?

**Comment by KrapivinAndrey** [comment]
@Pr-Mex Да. Только что попробовал. Зацепился по vnc. Окна я вижу но по итогу черный скриншот

**Comment by Pr-Mex** [comment]
@KrapivinAndrey 
Ок. Можешь приложить тест и базу? Я на своей убунте проверю.

**Comment by KrapivinAndrey** [comment]
Попробую облегченный вариант собрать

**Comment by EtherialSpirit** [comment]
Всем привет. Натолкнулись на похожую проблему. Есть ли решение?

**Comment by Pr-Mex** [comment]
@EtherialSpirit 

Уажите ваше окружение.

**Comment by Hapensky** [comment]
Привет! Аналогичная проблема - с помощью компоненты делаются темные (черные) скриншоты.
ВерсияVA: ver 1.2.041.37 (так же и на 1.2.042.1)
ТипКлиентаТестирования: Тонкий
ВерсияПлатформы: 8.3.24.1548
ТипПлатформы: Linux x86-64 (убунту)
РежимСовместимостиTestManager: НеИспользовать
есть менеджер окон openbox, запускается дисплей

Без VAExt скрины делаются нормально.

![Image](https://github.com/user-attachments/assets/cf436335-d27b-4ae2-8673-8bf5b5501c9c)

**Comment by Pr-Mex** [comment]
@Hapensky 
Думаю стоит провести эксперимент и поставить xfce - проблема уйдет?

**Comment by Hapensky** [comment]
@Pr-Mex, привет! 
Сделали образ с xfce, проверил, проблема осталась

**Comment by Pr-Mex** [comment]
@Hapensky 
Попробуйте поставить обычный дистрибутив убунты локально у себя в виртуалке и проверьте будут ли там сниматься скриншоты?

**Comment by beefree86** [comment]
@Pr-Mex Леонид, здравствуйте. Мы сейчас проверяем варианты с компонентой, у нас есть прогресс, скрины стали нормальными. Еще протестируем и напишем результат

**Comment by Hapensky** [comment]
Провели более углубленный анализ. Удалось разобраться. Проблема была в глубине цвета, стоял 16 бит, сделали 24. И на опенбоксе, и xfce теперь порядок. Но есть нюанс на опенбоксе, наезжает поверх тест клиента менеджер тестирования и тут не важно с компонентой или без. И какой-то рандом с этим, может заслонить, может нет, возможно такой эффект появляется, если нет модальных окон.

![Image](https://github.com/user-attachments/assets/2aa9618e-aa56-47a4-bd51-1eaeab8151be)

**Comment by beefree86** [comment]
А вот и результат )

**Comment by Pr-Mex** [comment]
@Hapensky 
@beefree86 
Понял, спасибо.
